pkgs/development/interpreters/zenroom/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "zenroom";
+  version = "3.5.0";
+
+  src = fetchurl {
+    url = "https://files.dyne.org/zenroom/nightly/zenroom-linux-amd64";
+    hash = "sha256-9xhQPrez4pk1zCzCyBqmNF/mkfChcA38p/s8//AwpZ8=";
+  };
```

Comment:
Why aren't you fetching the release from github? https://github.com/dyne/Zenroom

Reply:
I tried to use that at first, but I had a lot of issues with building it using cmake

Reply:
Do I need to do something in order to get this merged?